dbc_extract3/dbc/generator.py

```diff
@@ -2029,6 +2032,10 @@ def initialize(self):
         self._data_store.link('ItemSetSpell', 'id_item_set', 'ItemSet', 'bonus')
 
         self._data_store.link('ItemEffect', self._options.build < 25600 and 'id_item' or 'id_parent', self._options.build < 23436 and 'Item-sparse' or 'ItemSparse', 'spells')
+
+        if self._options.build >= dbc.WowVersion(9, 0, 0, 0):
+            self._data_store.link('SpellTargetRestrictions', 'id_parent', 'SpellName', 'targeting')
+
```

Comment:
If you go by the new data model stuff (see comment below where you add the field to spell_data_t record output), explicitly linking stuff into a parent db2 file is unnecessary, **if** the current format file defines this linkage (i.e., the `parent` value for `SpellTargetRestrictions` is set to `SpellName` in the format file).